apis/data/v0alpha1/client.go

```diff
@@ -0,0 +1,66 @@
+package v0alpha1
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data/utils/jsoniter"
+)
+
+type QueryDataClient interface {
+	QueryData(ctx context.Context, req QueryDataRequest, headers ...string) (int, *backend.QueryDataResponse, error)
+}
+
+type simpleHTTPClient struct {
+	url     string
+	client  *http.Client
+	headers []string
+}
+
+func NewQueryDataClient(url string, client *http.Client, headers ...string) QueryDataClient {
```

Comment:
I am not sure I understand the purpose of this new client? Is it an example?

Reply:
it may make sense in its own package (and perhaps with a generic for the query type) but given this is all in experimental/v0alpha I don't think it matters too much yet